packages/jest-worker/src/Queue.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {QueueChildMessage} from './types';
+
+export default class Queue {
+  private data: Array<QueueChildMessage | null>;
+  private offset: number;
+
+  constructor() {
+    this.data = [];
+    this.offset = 0;
+  }
+
+  public push = (item: QueueChildMessage) => this.data.push(item);
```

Comment:
```suggestion
  push(item: QueueChildMessage) {
    this.data.push(item)
  }
```

somewhat same for the others - no need for the `public` modifier and don't use arrow functions

Reply:
I will update it now